scripts/zones/Southern_San_dOria/npcs/Varchet.lua

```diff
@@ -18,7 +18,6 @@ local GAME_TIE = 3;
 -----------------------------------
 
 function onTrade(player,npc,trade)
-
     if (npcUtil.tradeHas(trade, nil, 5)) then
```

Comment:
Only a handful of the npc utils wrappers are really useful. When it makes sense to use them, do. When you have to ask, that's a sign it was one of the crappy ones, kill with fire.